openrtb_ext/imp_appnexus_Keywords_test.go

```diff
@@ -0,0 +1,35 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKeywordsUtilUnmarshalJSON(t *testing.T) {
```

Comment:
**Nitpick:** Rename this to  `TestKeywordsUnmarshalJSON`

Reply:
Done in [034e995](https://github.com/prebid/prebid-server/pull/2649/commits/034e9950d26f5223e94b5341f8333f77cb32dbe6)